docs/compliance/reference/800-53/index.md

```diff
@@ -33,4 +33,4 @@ In addition, each control is assigned one or more of the following implementatio
 |Planned|The control is not in place. A plan for achieving full implementation should be included in the Plan of Action & Milestone documentation|
 |None|The control is not applicable within the environment. A description of why the requirement does not apply should be included|
 
-Control narratives that include an `[Assignment: ...]` block should be substituted by your organization's requirements or by the FedRAMP requirements.
+Control narratives that include an `[Assignment: ...]` block should be substituted by your organization's requirements or by an alternative set of requirements (e.g. FedRAMP baselines).
```

Comment:
avoid "e.g." "For instance" is fine, IMHO

Reply:
Updating